pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <artifactId>mod-inventory-storage</artifactId>
   <groupId>org.folio</groupId>
-  <version>19.3.0-SNAPSHOT</version>
+  <version>19.4.0-SNAPSHOT</version>
```

Comment:
As this is the start of a long lived branch used for patches, I don't think this is the next next version, rather it should be `19.3.1`

Reply:
Done